platform-emf/src/main/java/com/softicar/platform/emf/attribute/EmfInheritedAttribute.java

```diff
@@ -98,6 +98,12 @@ public IStaticObject getTestMarker() {
 		return originalAttribute.getTestMarker();
 	}
 
+	@Override
+	public boolean isScope() {
+
+		return getTable().getScopeField().map(it -> it.equals(baseField)).orElse(false);
```

Comment:
uh, that's wrong, strange that this compiles, it must be
```suggestion
		return originalAttribute.isScope();
```